src/common/FileSystem.cpp

```diff
@@ -124,6 +124,7 @@ namespace FS {
 
 #ifdef BUILD_ENGINE
 static Cvar::Cvar<bool> fs_legacypaks("fs_legacypaks", "Also load pk3s, ignoring version.", Cvar::NONE, false);
+static Cvar::Cvar<int> fs_maxSymlinkDepth("fs_maxSymlinkDepth", "Max depth of symlinks in zip paks. 0 means disabled.", Cvar::NONE, 0);
```

Comment:
The trailing dot in `fs_legacypaks` description string was a mistake of mine, the common usage is to not add trailing dot to cvar descriptions. Can you fix `fs_maxSymlinkDepth` description by removing the dot?

You may add an extra commit to remove the trailing dot in `fs_legacypaks` description and even rename to `fs_legacyPaks` while you're at it (another mistake of mine).